commit ebad6f614f7ad33de688c00e95031fbe9453cb2d (refs/changes/58/5458/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-02-04T15:30:34+00:00 (8 months ago)
    
    TRITON-1092 sdc-system-tests "sdc-system-tests.tap" not in TAP format
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/runtests b/runtests
index 5c9b1f4..7bf2e97 100755
--- a/runtests
+++ b/runtests
@@ -66,7 +66,7 @@ NODEUNIT=./node_modules/nodeunit/bin/nodeunit
 
 # Options.
 opt_test_pattern=
-opt_reporter=default
+opt_reporter=tap
 while getopts "hf:r:" opt
 do
     case "$opt" in
